.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 10.x
+          cache: yarn
+
+      - run: yarn install
+      - run: yarn lint:hbs
+      - run: yarn lint:js
+
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 10.x
+          cache: yarn
+
+      - run: yarn install
+      - run: yarn test
+
+  try-scenarios:
+    name: ${{ matrix.try-scenario }}
+    runs-on: ubuntu-latest
+    needs: ['lint', 'test']
+
+    strategy:
+      fail-fast: false
+      matrix:
+        try-scenario:
+          - ember-lts-2.16
+          - ember-lts-2.18
```

Comment:
Same I think we can drop 2.16 and 2.18 WDYT?

Reply:
see above. one thing at a time.